packages/cosmwasm/src/cosmwasmclient.searchtx.spec.ts

```diff
@@ -47,8 +47,8 @@ describe("CosmWasmClient.searchTx", () => {
 
   beforeAll(async () => {
     if (wasmdEnabled()) {
-      const pen = await Secp256k1Pen.fromMnemonic(faucet.mnemonic);
-      const client = new SigningCosmWasmClient(wasmd.endpoint, faucet.address, (signBytes) =>
+      const pen = await Secp256k1Pen.fromMnemonic(alice.mnemonic);
```

Comment:
Yes, using different accounts in different packages that target a live chain is much nicer.